agent/agent_endpoint_test.go

```diff
@@ -5308,14 +5308,23 @@ func TestAgent_Token(t *testing.T) {
 			effective: tokens{master: "M"},
 		},
 		{
-			name:      "set master ",
+			name:      "set master",
```

Comment:
While we are at it we should change this too, I see a lot of other references that can be changed in the test file

Reply:
I believe the plan is to only focus on external references for now. We can cleanup all the internal stuff after the 1.11 release.

Reply:
Yep, exactly that 👍🏻 

I have a WIP branch that renames the struct fields etc. but in this case, we'd leave the test name the same.